static/styles/popovers.css

```diff
@@ -450,7 +449,7 @@ ul {
 
     .col-left {
         padding: 0 10px 0 0;
-        word-break: break-all;
+        overflow-wrap: anywhere;
```

Comment:
Would probably be better to use `brea-word` value for `overflow-wrap` instead of `anywhere`. The [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility) mention that `anywhere` is not supported by Safari browser. And the behavior should be same for both values since we are not setting width to `min-content` in this case.